enterprise/server/crypter/crypter.go

```diff
@@ -23,6 +23,8 @@ const (
 	// key-derivation function in crypter_service.
 	EncryptedDataHeaderVersion = 1
 
+	PlainTextChunkSize = 1024 * 1024 // 1 MiB
```

Comment:
nit: add a comment (ideally with enough info so that someone could change this and know the impact of doing so?)

Reply:
Done